Library/Homebrew/gpg.rb

```diff
@@ -5,7 +5,7 @@ def self.find_gpg(executable)
     which_all(executable).detect do |gpg|
       gpg_short_version = Utils.popen_read(gpg, "--version")[/\d\.\d/, 0]
       next unless gpg_short_version
-      Version.create(gpg_short_version.to_s) == Version.create("2.0")
+      Version.create(gpg_short_version.to_s) == Version.create("2.0") || Version.create("2.1")
```

Comment:
Personally, I think it might be better to leave it hardcoded to check for the two specific version branches. Currently, and by deliberate design, this finds any executable of the specified version(s) in the PATH, not just ones installed by Homebrew. This is behavior we probably want to keep.

However, if it were simplified to `>= Version.create("2.0")`, once the new 2.2 branch is released in the near future it would be be picked up as well. This would probably be problematic because, for one, I doubt that all the formulae that use the gpg2 requirement would support the new version immediately. This would of course lead to issues installing and using those formulae.

Therefore, here I'd rather take the more conservative approach and not simplify it to a single comparison. This way we have more control over when and how the `GnuPG 2.2.x` branch is introduced into the Homebrew ecosystem.